tools/db_bench_tool.cc

```diff
@@ -2349,6 +2392,16 @@ class CombinedStats {
     return avg;
   }
 
+  double CalcConfidence95(std::vector<double> data) {
```

Comment:
Also, we should have a comment here mentioning that the samples are probably not from a normal distribution, but this should nevertheless provide useful, rough inferences.